src/cockpit/beiboot.py

```diff
@@ -387,14 +395,9 @@ def do_send_init(self):
         pass  # wait for the peer to do it first
 
     def do_init(self, message):
-        # https://github.com/cockpit-project/cockpit/issues/18927
-        #
         # We tell cockpit-ws that we have the explicit-superuser capability and
-        # handle it ourselves (just below) by sending `superuser-init-done` and
-        # passing {'superuser': False} on to the actual bridge (Python or C).
-        if isinstance(message.get('superuser'), dict):
-            self.write_control(command='superuser-init-done')
-        message['superuser'] = False
+        # handle it ourselves in SshPeer.
```

Comment:
I think superuser doesn't even need to be mentioned here anymore, does it?

Reply:
You mean dropping the whole comment? You are right. Perhaps it should instead say

> Forward our channel options to the remote bridge

? It's clear to me *right now*, but our future selves may wonder about it.

Reply:
Looks like this now locally:

> forward our init options to the remote bridge; we are transparent except for the explicit-superuser handling in SshPeer

Better?

Reply:
Pushed.